tracee-rules/engine/engine.go

```diff
@@ -102,12 +102,11 @@ func (engine *Engine) consumeSources(done <-chan bool) {
 					}
 					for _, sel := range se {
 						if sel.Source == "tracee" {
-							sig.OnSignal(types.SignalSourceComplete("tracee"))
+							sig.OnSignal(types.SignalSourceComplete("tracee")) // TODO: Handle error?
 							break
 						}
 					}
 				}
-				engine.inputs.Tracee = nil
```

Comment:
what's the motivation for this change? this line is preventing the containing loop from continuing infinitely and wasting cycles. I'm open to another solution but we need to be mindful of these changes

Reply:
Replied & fixed in another comment.